src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/services/WorkflowValidationService.scala

```diff
@@ -82,4 +83,14 @@ class WorkflowValidationServiceImpl @Inject()(override val workflowRepository: W
     }
     Future.successful(projectValidation.toSeq)
   }
+
+  private def validateWorkflowData(workflow: WorkflowJoined)(implicit ec: ExecutionContext): Future[Seq[ApiError]] = {
+    workflowRepository.getWorkflow(workflow.id)
+      .map(persistedWorkflowData => if (persistedWorkflowData == workflow) {
```

Comment:
Unfortunately it will not work. Check updateWorkflow in WorkflowService. Workflow that you receive contains no ids

Reply:
saw this one hence on effects I tried to grab all the ids. thanks.